_sass/_base.scss

```diff
@@ -0,0 +1,206 @@
+/**
```

Comment:
I think valid, taking into account the real objective of the project. The simpler the better!